Tools/Pipeline/Common/AssemblyAttributes.cs

```diff
@@ -81,6 +81,14 @@ public static string AssemblyCompany
 				return ((AssemblyCompanyAttribute)attributes[0]).Company;
 			}
 		}
+
+        public static string[] Developers
+        {
+            get
+            {
+                return new string[] { "cra0zy (Harry)", "dellis1972 (Dean Ellis)", "dneelyep (Daniel Neel)", "emarcotte (Eugene Marcotte)", "JamesLupiani (James Lupiani)", "jamesford42 (James Ford)", "KonajuGames (Steve 'Sly' Williams)", "mgarstenauer (Martin Garstenauer)", "nightst4r (Mark)", "nkast (Nikos Kastellanos)", "polsys (Petri Laarne)", "RayBatts (Ray Batts)", "Shqrdx (Yuri Roubinsky)", "thiagoabreu (Thiago 'Jedi' Abreu)", "tomspilman (Tom Spilman)" };
```

Comment:
@KonajuGames @dellis1972 - Do we want to allow this?

IMO we don't. If we allow this then why not a credit for any contribution made? What about every old commit in history? Where does this all go? When does it end?